### DISCUSSION TITLE
RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory

### DISCUSSION THREAD

**Discussion by forwiat** [fishaudio/Bert-VITS2#272] [Q&A] [answered]
When I load the model Erlangshen-MegatronBert-1.3B-Chinese from openi, An error occurred: RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory

**Comment by forwiat** [comment]
I just use such code:
```
import torch
torch.load("pytorch_model.bin")
```
It may be from the difference of pytorch.(I use torch==2.0.1). I wonder which version should I use. Thanks for answer:)

**Comment by Stardust-minus** [comment, marked as answer]
checkpoint is damaged.Try down load the model "inside" the download page:
![image](https://github.com/fishaudio/Bert-VITS2/assets/52093397/feb87e51-c83c-413b-9eaa-db3519e09a11)
click the upper button to download,rather than here:
![image](https://github.com/fishaudio/Bert-VITS2/assets/52093397/e523e8dc-d57c-4723-9a54-20b94c98f2a1)
Or,you can use openi pip module to download in cli.
Here is the openi module instruction to install it:
https://openi.pcl.ac.cn/docs/index.html#/api/intro
You still need to login,and run `openi model download` ,follow the info.
And finally,you still can download it on huggingface(if possible) :)

**Comment by forwiat** [comment]
Thanks for solution! I will try it again!
